web/cobrands/peterborough/assets.js

```diff
@@ -15,14 +32,45 @@ var defaults = {
             SRSNAME: "urn:ogc:def:crs:EPSG::3857"
         }
     },
-    max_resolution: 4.777314267158508,
-    geometryName: 'msGeometry',
-    srsName: "EPSG:3857",
-    body: "Peterborough City Council",
-    strategy_class: OpenLayers.Strategy.FixMyStreet
-};
+    geometryName: 'msGeometry'
+});
 
-fixmystreet.assets.add(defaults, {
+var arcgis_defaults = $.extend(true, {}, defaults, {
+    protocol_class: OpenLayers.Protocol.Peterborough,
+    format_class: OpenLayers.Format.GeoJSON,
+    http_options: {
+        params: {
+            inSR: '3857',
+            outSR: '3857',
+            f: 'geojson'
+        }
+    },
+    geometryName: 'SHAPE'
+});
+
+var waste_categories = ['General fly tipping', 'Hazardous fly tipping', 'Offensive graffiti', 'Non offensive graffiti' ];
+
+OpenLayers.Layer.PeterboroughVectorNearest = OpenLayers.Class(OpenLayers.Layer.VectorNearest, {
+    getNearest: function(lonlat) {
+        var point = new OpenLayers.Geometry.Point(lonlat.lon, lonlat.lat);
+        var feature = this.getFeatureAtPoint(point);
+        var radius = this.fixmystreet.nearest_radius || 10;
+        if ( OpenLayers.Util.indexOf(waste_categories, fixmystreet.reporting.selectedCategory().category) != -1) {
+            radius = 2;
```

Comment:
Is there a reason why this isn't just 2 for everything? Given the shapes are vectors, not lines, does it matter if it was 2 everywhere instead of 10? Wouldn't need this subclass at all then.